### PR TITLE
wormchain - contracts deploy CI test

### PR DESCRIPTION
**Pull request by justinschuldt**
Adds a shell script that starts up a local wormchain node and then deploys contracts and runs tests, so that it can be run in CI.
Adds the new job to CI.

### CODE REVIEW

**Comment by justinschuldt** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2296** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2296" target="_blank">Graphite</a> 
    * **PR #2297** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2297" target="_blank">Graphite</a> 
      * **PR #2298** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2298" target="_blank">Graphite</a> 
        * **PR #2299** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2299" target="_blank">Graphite</a> 
          * **PR #2300** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2300" target="_blank">Graphite</a> 
            * **PR #2301** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2301" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2301?utm_source=stack-comment).

**Comment by justinschuldt** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2296** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2296" target="_blank">Graphite</a> 
    * **PR #2297** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2297" target="_blank">Graphite</a> 
      * **PR #2298** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2298" target="_blank">Graphite</a> 
        * **PR #2299** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2299" target="_blank">Graphite</a> 
          * **PR #2300** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2300" target="_blank">Graphite</a> 
            * **PR #2301** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2301" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2301?utm_source=stack-comment).

**Review by conorpp — APPROVED** [review]

**Comment by justinschuldt** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: mPfcRG7PvNPnESsfPBuJ.
See details on [graphite.dev](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2301)

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -260,11 +260,23 @@ jobs:
...
         with:
           go-version: "1.19.3"
-      - run: |
+      - name: make proto -B && make test
+        run: |
           cd wormchain
           make proto -B
           git diff --name-only --exit-code && echo "✅ Generated proto matches committed proto" || (echo "❌ Generated proto differs from committed proto, run \`make proto -B\` and commit the result" >&2 && exit 1)
           make test
+<<<<<<< HEAD
```
Comment (evan-gray) [inline .github/workflows/build.yml:269]: Merge conflict committed?

**Review by evan-gray — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (justinschuldt) [reply .github/workflows/build.yml]: wow, did that with Graphite. Fixed.

**Comment by evan-gray** [comment]
Closing after discussion with @justinschuldt, this is adequately covered by accountant e2e in tilt.

**Pull request closed without merging**